ui/components/table.js

```diff
@@ -51,6 +51,10 @@ export default function Table({
   const [checkedAll, setCheckedAll] = useState(false)
   const [indeterminate, setIndeterminate] = useState(false)
 
+  const handleSelectedRowIds = ids => {
```

Comment:
```suggestion
  function handleSelectedRowIds (ids) {
```

This is completely a style choice (since we aren't referring to `this`). Regular function declarations feel more standard.